cmd/main.go

```diff
@@ -188,12 +188,12 @@ var createBackupCmd = &cobra.Command{
 		return initBackupProvider()
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
-		c, err := client.New(context.Background(), viper.GetString(serverAddrFlg))
+		c, err := client.New(cmd.Context(), viper.GetString(serverAddrFlg))
```

Comment:
Potentially interrupts the server when the client connection times out, I am not sure.